### ISSUE TITLE
Syntactically wrong altLabels in knowledge graph?

### ISSUE THREAD

**Issue by FNieschalk** [Cognonto/kko#2]
I noticed that in  https://github.com/Cognonto/kbpedia/blob/master/versions/1.60/kbpedia_reference_concepts_linkage_inferrence_extended.zip there are some altLabels stored in one literal, each seperated by `||`, e.g.: 

`skos:altLabel "Your Paintings collection identifier||BBC Your Paintings collection identifier||Art UK collection identifier"@en ;`

Whereas others are written in statements with multiple objects (literals): 

`skos:altLabel "apartments"@en , "flat"@en , "flats"@en ;`

As far as I know, only the second statement is valid N3, right? At least Apache Jena, into which I am reading the file, will only parse the second statement correctly (as 3 independent triples) but not the first.

This may also be some issue with Apache Jena... if so I am sorry to have bothered you.

**Comment by fgiasson** [comment]
@FranzNieschalk that is right. This is probably due to some issue in the generation code that doesn't extend those labels in the second form that you put in this issue (and which should). I will take a look at that a bit later, thanks for reporting!
